signer/tests/integration/postgres.rs

```diff
@@ -6138,6 +6138,42 @@ async fn writing_key_rotation_transactions() {
     testing::storage::drop_db(db).await;
 }
 
+mod p2p_peers {
+    use libp2p::{Multiaddr, PeerId};
+    use signer::testing::network::MultiaddrExt as _;
+
+    use super::*;
+
+    #[tokio::test]
+    async fn write_read_p2p_peer() {
+        let db = testing::storage::new_test_database().await;
+        let mut rng = get_rng();
+
+        let pub_key: PublicKey = Faker.fake_with_rng(&mut rng);
+        let peer_id: PeerId = pub_key.into();
+        let multiaddr = Multiaddr::random_memory();
+        let utc_now = time::OffsetDateTime::now_utc();
+
+        db.update_peer_connection(&pub_key, &peer_id, multiaddr.clone())
+            .await
+            .expect("Failed to insert peer connection");
+
+        let peers = db.get_p2p_peers().await.unwrap();
+        assert_eq!(peers.len(), 1);
+        assert_eq!(*peers[0].peer_id, peer_id);
+        assert_eq!(peers[0].public_key, pub_key);
+        assert_eq!(*peers[0].address, multiaddr);
+        // Ensure that the last_dialed_at timestamp is within a reasonable
+        // timespan from utc_now.
+        assert!(*peers[0].last_dialed_at - utc_now < time::Duration::seconds(5));
+
+        dbg!(&peers);
+        dbg!(&peers[0].last_dialed_at);
```

Comment:
nit: leftover `dbg!`

Reply:
Took care of that one in [8f88f86](https://github.com/stacks-sbtc/sbtc/pull/1689/commits/8f88f86849cd9a7e9eb5a07ae64664ea2c7131c6)